### ISSUE TITLE
See the latest published versions on a task

### ISSUE THREAD

**Issue by Innders** [ynput/ayon-frontend#341]
### Stories

1. An artist wants to see the latest version they just published.
2. A sup wants to know if a version has addressed some previous feedback.

### Solutions

- When a new version is published, create an new activity of `type=version/publish`
- Published versions appear in the activity feed amongst other comments and activities. This gives a visual timeline of actions taken on the task.
- Versions must be clickable to be able to view that version.

![Image](https://github.com/ynput/ayon-frontend/assets/49156310/ca60a7fc-4b33-4254-a818-046f65a8fa1f)

### Questions

- [x] A version can have multiple representations, which one/s do we show?
- [x] Activity type name? `version` `publish`
- [x] If multiple products are published at the same time can they be grouped as one activity, should that be BE or FE job?
- [x] What happens when 20 new versions are published at once, are some hidden to prevent the feed from becoming flood? Which ones are shown first? `lighting renders beauty and then 20 AOV layers`.

### TODO

- [x] Design
- [x] Activity type creation from publish event (server)
- [x] Frontend component

**Comment by Innders** [comment]
I believe this should be moved to the Web Viewer EPIC as it's very interlinked with reviewing versions.

These versions should be clickable and when they are clicked Web Viewer is needed.

Is it worth having these versions in the feed if they are not clickable, it will users first instinct to click them. I feel it would be a half baked feature.

**Comment by mkolar** [comment]
@dee-ynput what do you think? I'd say I agree with @Innders we've expanded the scope of the epic on a few other fronts, it would be reasonable to move this over.

**Comment by dee-ynput** [comment]
I agree that the feed UI shouldn't become a review UI.
We must think of it as a side panel for the upcoming player/viewer.
In this regard, a clickable entry seems appropriate. Maybe a tooltip with a thumbnail if available.

The foreseen issue is when a task is generating many products.
Let's implement something simple which covers most cases (a few products in the task), and later on we'll be able to design a UX for special cases.

This will be easier to get right once we have the Player/Viewer and more practice with the feed.

Tl;Dr let's wait for the problem before fixing it 😌

**Comment by Innders** [comment]
> The foreseen issue is when a task is generating many products.

This is an issue that needs to be worked out, there is no simple implementation.

I strongly believe this should be moved Web Viewer as it interlinks so tightly with it. Web Viewer is fairly small in scope and so it wouldn't be too much more to do this there.
